linkchecker

```diff
@@ -547,22 +725,32 @@ else:
 if do_profile:
     if has_profile:
         if os.path.exists(_profile):
-            print(_("""Overwrite profiling file %(file)r?
-Press Ctrl-C to cancel, RETURN to continue.""") % {"file": _profile})
+            print(
+                _(
+                    """Overwrite profiling file %(file)r?
+Press Ctrl-C to cancel, RETURN to continue."""
```

Comment:
```suggestion
                    "Overwrite profiling file %(file)r?\n"
                    "Press Ctrl-C to cancel, RETURN to continue."
```

(I am assuming xgettext or whatever is used to extract translations from the source files will see these strings as the same, because they are)

Reply:
Done.